tests/data_access.rs

```diff
@@ -28,7 +28,7 @@ struct Loaded {
 }
 
 const TRACKABLE_MAX: u64 = 3600 * 1000 * 1000;
-const SIGFIG: u32 = 3;
+const SIGFIG: u8 = 3;
```

Comment:
I added a comment to the `SIGFIG`s in tests, but searching for that led me to the benchmarks, which I haven't been building. Criterion is, ah, "lightly documented". How should I run those bad boys?